catalog/MDCCatalog/MDCCatalogCollectionViewCell.swift

```diff
@@ -20,22 +20,29 @@ import MaterialComponents.MaterialTypography
 
 class MDCCatalogCollectionViewCell: UICollectionViewCell {
 
-  var label = UILabel()
-  let pad = CGFloat(14)
-  let tile = MDCCatalogTileView(frame: CGRect.zero)
+  fileprivate struct Constants {
+    static let imageWidthHeight: CGFloat = 80
+    static let xPadding: CGFloat = 14
+    static let yPadding: CGFloat = 16
+  }
```

Comment:
I wonder if it's beneficial to align our work with motion specs here as well and call this structure the "Spec" ?

Reply:
I don't think so in this cast. These are layout constants and they aren't defined by the spec but rather by our app's designer.

Reply:
Let's discuss offline - no need to block this PR!

Reply:
👌